packages/ui/origin-ui-core/src/components/certificates/Inbox/InboxItem.tsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import { Checkbox, Button } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { useOriginConfiguration } from '../../../utils/configuration';
+import { formatDate, LightenColor, moment, useTranslation } from '../../../utils';
+import { DeviceIcon } from '../../DeviceIcon';
+import { CertificateSource } from '../../../features/certificates';
+
+export interface IInboxItemData {
+    id: string;
+    name: string;
+    country: string;
+    type: string;
+    capacity: number;
+    certificates: IInboxCertificateData[];
+}
+
+export interface IInboxCertificateData {
+    id: string;
+    dateStart: number;
+    dateEnd: number;
+    energy: number;
+    maxEnergy: number;
+    source: CertificateSource;
+    assetId: string;
+}
+
+export function InboxItem(props: {
+    device: IInboxItemData;
+    selected: string[];
+    selectedDevices: string[];
+    onDeviceSelect: (id: string) => void;
+    onCertificateSelect: (id: string, deviceId: string) => void;
+}): JSX.Element {
+    const { device, selected, onDeviceSelect, onCertificateSelect, selectedDevices } = props;
+    const configuration = useOriginConfiguration();
+
+    const { MAIN_BACKGROUND_COLOR, SIMPLE_TEXT_COLOR } = configuration?.styleConfig;
+
+    const useStyles = makeStyles({
+        device: {
+            padding: '18px 20px',
+            background: LightenColor(MAIN_BACKGROUND_COLOR, 4),
+            boxShadow: '0 2px 4px rgba(0,0,0,.2)',
+            display: 'flex',
+            flexDirection: 'row'
+        },
+
+        verticalCenter: {
+            display: 'flex',
+            alignItems: 'center'
+        },
+
+        entry: {
+            marginBottom: '10px'
+        },
+
+        certificate: {
+            padding: '18px 20px',
+            background: LightenColor(MAIN_BACKGROUND_COLOR, 6),
+            boxShadow: '0 2px 4px rgba(0,0,0,.2)',
+            display: 'flex',
+            flexDirection: 'row',
+            borderBottom: '1px solid #464646'
+        },
+
+        selected: {
+            background: '#362C45'
```

Comment:
Don't use hard-coded colors - they will cause issues, when we will switch the color scheme